# HG changeset patch
# User Rik <rik@octave.org>
# Date 1409344056 25200
#      Fri Aug 29 13:27:36 2014 -0700
# Node ID 13fa7b5375a90880fce92df9a45ad3c0b454d13c
# Parent  bbd2cee3da788cab273408ce5781e42c48c12dcd
__unimplemented__.m: Update list of unimplemented functions.

* __unimplemented__.m: Update list of unimplemented functions.

diff --git a/scripts/help/__unimplemented__.m b/scripts/help/__unimplemented__.m
--- a/scripts/help/__unimplemented__.m
+++ b/scripts/help/__unimplemented__.m
@@ -529,21 +529,24 @@ endfunction
 
 function list = missing_functions ()
   persistent list = {
   "MException",
   "RandStream",
   "Tiff",
   "VideoReader",
   "VideoWriter",
+  "addCause",
+  "addcats",
   "align",
   "alim",
   "alpha",
   "alphamap",
   "annotation",
+  "array2table",
   "audiodevinfo",
   "audioinfo",
   "audioplayer",
   "audioread",
   "audiorecorder",
   "audiowrite",
   "bar3",
   "bar3h",
@@ -567,132 +570,155 @@ function list = missing_functions ()
   "campan",
   "campos",
   "camproj",
   "camroll",
   "camtarget",
   "camup",
   "camva",
   "camzoom",
+  "categorical",
+  "categories",
   "cdf2rdf",
   "cdfepoch",
   "cdfinfo",
+  "cdflib",
   "cdfread",
   "cdfwrite",
+  "cell2table",
   "cellplot",
-  "checkin",
   "checkcode",
+  "checkin",
   "checkout",
-  "cholinc",
   "clearvars",
   "clipboard",
   "cmopts",
   "colordef",
   "colormapeditor",
   "commandhistory",
   "commandwindow",
   "condeig",
   "coneplot",
   "containers.Map",
   "contourslice",
+  "corrcoef",
+  "countcats",
   "createClassFromWsdl",
   "createSoapMessage",
   "customverctrl",
   "datacursormode",
   "dbmex",
   "dde23",
   "ddeget",
   "ddensd",
   "ddesd",
   "ddeset",
   "decic",
   "delaunayTriangulation",
   "depdir",
   "depfun",
+  "details",
   "deval",
   "dialog",
   "dither",
   "docsearch",
   "dragrect",
   "dynamicprops",
   "echodemo",
+  "empty",
+  "enumeration",
   "evalc",
+  "events",
   "export2wsdlg",
   "figurepalette",
   "filebrowser",
   "fill3",
+  "findprop",
   "fitsdisp",
   "fitsinfo",
   "fitsread",
   "fitswrite",
+  "flip",
   "flow",
   "frame2im",
   "freqspace",
   "funm",
   "gammaincinv",
+  "getReport",
   "getframe",
   "getpixelposition",
   "gobjects",
   "grabcode",
   "graymon",
   "griddedInterpolant",
   "gsvd",
   "guide",
-  "guihandles",
-  "handle",
   "h5create",
   "h5disp",
   "h5info",
   "h5read",
   "h5readatt",
   "h5write",
   "h5writeatt",
+  "handle",
   "hdfinfo",
   "hdfread",
+  "height",
   "hgexport",
   "hgsetget",
   "hgtransform",
   "im2frame",
   "im2java",
   "imapprox",
   "import",
   "inmem",
+  "innerjoin",
   "inspect",
-  "instrcallback",
-  "instrfind",
-  "instrfindall",
   "integral",
   "integral2",
   "integral3",
   "interpstreamspeed",
-  "iscom",
-  "isinterface",
+  "isKey",
+  "iscategorical",
+  "iscategory",
+  "ismissing",
   "isocaps",
+  "isordinal",
+  "isprotected",
   "isstudent",
+  "istable",
+  "isundefined",
+  "isvalid",
+  "javaMethodEDT",
+  "javaObjectEDT",
+  "join",
+  "keys",
+  "last",
   "ldl",
   "libfunctions",
   "libfunctionsview",
   "libisloaded",
   "libpointer",
   "libstruct",
   "light",
   "lightangle",
   "lighting",
   "linkdata",
   "listfonts",
   "loadlibrary",
+  "localfunctions",
   "lscov",
   "lsqr",
   "makehgtform",
   "material",
   "matfile",
   "matlabrc",
   "memmapfile",
   "memory",
-  "metaclass",
+  "mergecats",
   "methodsview",
   "minres",
   "mlintrpt",
   "mmfileinfo",
   "movegui",
   "movie",
   "movie2avi",
   "multibandread",
@@ -701,18 +727,20 @@ function list = missing_functions ()
   "nccreate",
   "ncdisp",
   "ncinfo",
   "ncread",
   "ncreadatt",
   "ncwrite",
   "ncwriteatt",
   "ncwriteschema",
+  "netcdf",
   "noanimate",
   "notebook",
+  "notify",
   "ode113",
   "ode15i",
   "ode15s",
   "ode23",
   "ode23s",
   "ode23t",
   "ode23tb",
   "ode45",
@@ -721,111 +749,128 @@ function list = missing_functions ()
   "odextend",
   "open",
   "openfig",
   "opengl",
   "openvar",
   "ordeig",
   "ordqz",
   "ordschur",
+  "outerjoin",
   "padecoef",
   "parseSoapResponse",
   "pathtool",
   "pcode",
   "pdepe",
   "pdeval",
   "plotbrowser",
   "plotedit",
   "plottools",
   "printdlg",
   "printopt",
   "printpreview",
   "profsave",
   "propedit",
+  "properties",
   "propertyeditor",
   "psi",
   "publish",
   "qmr",
   "quad2d",
   "rbbox",
+  "readtable",
   "reducepatch",
   "reducevolume",
-  "readasync",
+  "remove",
+  "removecats",
+  "renamecats",
+  "reordercats",
   "rng",
   "rotate",
+  "rowfun",
+  "savefig",
   "scatteredInterpolant",
   "selectmoveresize",
   "sendmail",
   "serial",
-  "serialbreak",
   "setpixelposition",
   "showplottool",
   "smooth3",
   "snapnow",
   "sound",
   "soundsc",
   "ss2tf",
+  "stack",
+  "standardizeMissing",
   "startup",
-  "stopasync",
   "stream2",
   "stream3",
   "streamline",
   "streamparticles",
   "streamribbon",
   "streamslice",
   "streamtube",
-  "strings",
+  "struct2table",
   "subvolume",
+  "summary",
   "superclasses",
   "surf2patch",
+  "sylvester",
   "symmlq",
-  "syntax",
+  "table2array",
+  "table2cell",
+  "table2struct",
   "texlabel",
   "textwrap",
   "tfqmr",
+  "throw",
+  "throwAsCaller",
+  "timeit",
   "timer",
   "timeseries",
   "todatenum",
   "toolboxdir",
   "triangulation",
   "tscollection",
-  "tstool",
   "uibuttongroup",
-  "uicontextmenu",
-  "uicontrol",
   "uigetpref",
   "uiimport",
   "uiopen",
-  "uipanel",
-  "uipushtool",
   "uiresume",
   "uisave",
   "uisetcolor",
   "uisetfont",
   "uisetpref",
   "uistack",
   "uitable",
   "uitoggletool",
   "uitoolbar",
   "uiwait",
   "undocheckout",
   "unicode2native",
   "unloadlibrary",
   "unmesh",
+  "unstack",
   "userpath",
+  "values",
+  "varfun",
+  "verLessThan",
   "verctrl",
-  "verLessThan",
   "viewmtx",
   "visdiff",
   "volumebounds",
   "web",
-  "whatsnew",
+  "width",
   "winopen",
   "winqueryreg",
   "workspace",
+  "writetable",
+  "xlsfinfo",
+  "xlsread",
+  "xlswrite",
   "xmlread",
   "xmlwrite",
   "xslt",
   "zoom",
   };
 endfunction
 
 
